src/app/auth/auth.component.html

```diff
@@ -3,14 +3,14 @@
   <!-- Holder window -->
   <div class="min-h-md w-full max-w-md my-5 mx-2 px-4 py-8 bg-white shadow-md">
     <!-- Upper Holder - contains Logo -->
-    <div class="w-full h-2/6 rounded-t-md flex justify-center items-center">
-      <img src="../../assets/default-logo.png" class="w-30" alt="Logo image">
+    <div class="w-full px-12 h-2/6 rounded-t-md flex justify-center items-center">
+      <img src="../../assets/default-logo.png" alt="Logo image">
     </div>
 
     <!-- Bottom Holder Screen - Contains login options -->
     <div class="w-full h-4/6 bg-white rounded-b-md px-8 flex items-center">
       <div class="flex flex-col w-full">
-        <div class="flex content-center justify-center items-center">
+        <!-- <div class="flex content-center justify-center items-center">
           <label for="network" class="">
             Network
           </label>
```

Comment:
Again, undocumented commented code.

but, nvm, this component will be refactored.